app/editor/src/features/admin/sources/services/ServiceIngestSettings.tsx

```diff
@@ -81,23 +79,23 @@ export const ServiceIngestSettings: React.FC<IServiceIngestSettingsProps> = () =
             options={dataLocations}
             required
           />
-          {showImportContent && (
+          <FormikSelect
```

Comment:
Now that streams can generate content (clips) we need to display these settings.